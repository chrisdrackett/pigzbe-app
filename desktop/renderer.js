import {AppRegistry} from 'react-native';
import App from './bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
