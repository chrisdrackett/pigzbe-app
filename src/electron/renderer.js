import {AppRegistry} from 'react-native';
// import App from '../../App';
import App from '../../dist/bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
