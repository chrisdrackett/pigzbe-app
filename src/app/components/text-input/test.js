import React from 'react';
import renderer from 'react-test-renderer';
import TextInput from './';

const props = {
    label: 'Label',
    onPress: () => {},
    plain: false
};

describe('TextInput', () => {
    test('renders correctly', () => {
        const tree = renderer.create(<TextInput {...props}/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
