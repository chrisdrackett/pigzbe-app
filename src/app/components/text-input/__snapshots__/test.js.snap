// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextInput renders correctly 1`] = `
<TextInput
  allowFontScaling={true}
  onChangeText={[Function]}
  placeholder={undefined}
  placeholderTextColor="#97abc7"
  style={
    Object {
      "alignSelf": "stretch",
      "borderColor": "#ffffff",
      "borderRadius": 22.5,
      "borderWidth": 1,
      "color": "#ffffff",
      "fontFamily": "Poppins",
      "fontSize": 14,
      "fontWeight": "bold",
      "height": 45,
      "marginBottom": 10,
      "paddingLeft": 30,
      "paddingRight": 20,
    }
  }
  value=""
/>
`;
